### PR TITLE
add help command.

### PR DESCRIPTION
**Pull request by fennecdjay**
again.
But this time against your HEAD.
sorry for the previous help PR.

### CODE REVIEW

File: zpm.c [hunk 1]
```diff
@@ -236,9 +236,15 @@ int plugins_update_local_clone() {
...
+void usage() {
+    printf("%s\n", "Usage:\n\tzpm 'zsh-users/zsh-syntax-highlighting'");
+    printf("%s\n", "\nAvailable commands:\n\tzpm reset\n\tzpm list");
+    printf("%s\n", "\tzpm update\n\tzpm help");
+}
+
 int main(int argc, char* argv[]) {
     if (argc <= 1) {
-        printf("%s\n", "Usage:\n\tzpm 'zsh-users/zsh-syntax-highlighting'");
+		usage();
```
Comment (desyncr) [inline zpm.c]: Use (four) spaces rather than tabs for indentation.

**Review by desyncr — CHANGES_REQUESTED** [review]
Everything's looking good. One small nitpick though.

**Comment by desyncr** [comment]
Yay! Thank you so much @fennecdjay !

**Comment by fennecdjay** [comment]
sorry for the *nitpick* (I didn't know this word). I'm glad I'm figuring out better the *feature branching* stuff.
If you provide more info on some issues, I'll be happy to help!!!

**Comment by desyncr** [comment]
@fennecdjay Awesome! If you are up to it I'd be more than happy to add you as a zpm-zsh's collaborator. What that means is you'll have `push` access to the repository (be able to commit and merge to `master`) \*.

There are a few issues around and the TODO list for missing features. There is also [this](https://trello.com/b/TtXA4Ajz/zpm-zsh) Trello board for the project roadmap. Any idea and suggestion is greatly welcomed!

What do you think? Looking forward to it!

\* _Although, I usually work on `feature` branches and creating a PR for review, iterate on the given feedback and then merge the PR._

**Comment by fennecdjay** [comment]
I'll d be glad to help !!!!

P.S.: it looks like the trello board for zpm-zsh is private, maybe you have to invite me or something like that for me to see it.

**Comment by desyncr** [comment]
@fennecdjay Awesome. Trello board should be public now, I'll need you trello user so you're able to contribute to the board.

**Comment by fennecdjay** [comment]
I've found the board, and I think you are now invited on [this](https://trello.com/b/nTRU3qpX/meet-people). this is just a stub, but maybe you can get my user from there.

... I'm also very new to trello!!!

**Comment by desyncr** [comment]
@fennecdjay Added to the board! Welcome!

**Comment by fennecdjay** [comment]
Thanks!

**Pull request merged**